app/dao/service_callback_dao.py

```diff
@@ -0,0 +1,25 @@
+from notifications_utils.statsd_decorators import statsd
+from sqlalchemy.orm.exc import NoResultFound
+from app.models import (ServiceCallback)
+
+
+@statsd(namespace="dao")
+def dao_get_callback_include_payload_status(service_id, service_callback_type):
+    # Throw error is you return 0 or more than 1 row
+    try:
+        row = ServiceCallback\
+            .query.filter_by(service_id=service_id)\
+            .filter_by(callback_type=service_callback_type) \
+            .filter_by(include_provider_payload=True) \
+            .one()
+
+        include_provider_payload = row.include_provider_payload
+
+    except (NoResultFound, AttributeError) as e:
+        raise e
```

Comment:
There's no point to an except block that just re-raises.  You can delete it and the "try" above.